CONTRIBUTING.md

```diff
@@ -24,7 +32,7 @@
       * `sbt nmesos-cli/universal:packageZipTarball`
     * Publish the new version in a public repo ...
       * `sbt nmesos-cli/universal:publish`
-  * (optional) Update the `brew` [Formula](Formula/nmesos-cli.rb)
+  * Update the `brew` [Formula](Formula/nmesos-cli.rb)
     * Modify url to the new release and also replace the sha with the new sha (by running `shasum -a 256 cli/target/universal/nmesos-cli-0.2.19.tgz`)
   * Commit to `master`
-* Note: If something goes wrong and you need to re-release you first need to delete the old file (with `curl -X DELETE https://nmesos-releases.s3-eu-west-1.amazonaws.com/nitro-public/repo/nitro/nmesos-cli/0.2.19/nmesos-cli-0.2.19.tgz`).
+* Note: If something goes wrong and you need to re-release you first need to delete the old file (with `aws --profile nmesos s3 rm s3://nmesos-releases.s3-eu-west-1.amazonaws.com/nitro-public/repo/nitro/nmesos-cli/0.2.19/nmesos-cli-0.2.19.tgz`).
```

Comment:
Now that we have limited the write access curl will (obviously/rightfully) not work anymore.